### DISCUSSION TITLE
Export .Spoke to be used in A-Frame

### DISCUSSION THREAD

**Discussion by qiangjiadong** [mozilla/hubs#3553] [Spoke Questions]
May I ask if there is a way to export everything from one .spoke scene and use it into A-Frame via HTML?

It seems export GLTF from Spoke only works for meshes, particles effects and 2D images etc are not included.

Any other suggestions on export? Thks!!!

**Comment by pattersonbl2** [comment]
are you still having issues with this ?

**Comment by Seikida** [comment]
Hello,

any solutions ?

**Reply by pattersonbl2** [reply]
We haven't found any solution yet.  This question would be great to also ask in our community discord. You can join here (https://discord.com/invite/dFJncWwHun). I will ask my team if they may have a quick answer as well.
